orchagent/vxlanorch.h

```diff
@@ -117,18 +180,44 @@ class VxlanTunnel
     void incNextHopRefCount(IpAddress& ipAddr, MacAddress macAddress, uint32_t vni);
     void decNextHopRefCount(IpAddress& ipAddr, MacAddress macAddress, uint32_t vni);
 
+    bool deleteMapperHW(uint8_t mapper_list, tunnel_map_src_t map_src);
+    bool createMapperHW(uint8_t mapper_list, tunnel_map_src_t map_src);
+    bool createTunnelHW(uint8_t mapper_list, tunnel_map_src_t map_src, bool with_term = TRUE);
+    bool deleteTunnelHW(uint8_t mapper_list, tunnel_map_src_t map_src, bool with_term = TRUE);
```

Comment:
should this value  `bool with_term = TRUE`  be `bool with_term = true` ??

Reply:
yes. will change